.github/workflows/test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: pnpm typecheck
 
       - name: Lint
-        run: pnpm lint
+        run: pnpm lint:ci
```

Comment:
biome CLI has a command called `biome ci` which runs the formatter, linter and the import sorter in read-only mode meaning, it won't change any files at all which is what I used here for CI